### ISSUE TITLE
Version 24.0 not acting as a reply

### ISSUE THREAD

**Issue by davidmead** [dshanske/indieweb-post-kinds#59]
Hi @dshanske.

I just updated to the latest version. When I paste the URL in the 'Post Properties' and hit retrieve I now get a dialog box with "davidjohnmead.com says:" and an OK button.

<img width="632" alt="screenshot 2016-07-22 22 11 03" src="https://cloud.githubusercontent.com/assets/1266930/17075202/136d2e46-505a-11e6-8bb8-3bca37109cef.png">

I click OK. Bridgy Publish is set to Twitter and Post Kinds is set to reply. When I publish the tweet doesn't appear in my timeline at all, and [the post](http://djm.click/blog/b/1wd) doesn't show the content of what I'm replying to, but does show the new icon.

<img width="684" alt="screenshot 2016-07-22 22 21 19" src="https://cloud.githubusercontent.com/assets/1266930/17075220/a7adccf0-505a-11e6-8a7e-571283903592.png">

Any ideas?

**Comment by dshanske** [comment]
I haven't tested with Twitter in a few revisions. Try another site... and see and I will experiment.

**Comment by davidmead** [comment]
I tried replying to one of my own posts @dshanske. Doing it in stages:

1. Pasted [URL of existing post](http://davidjohnmead.com/blog/2016/07/25/7024/) into 'Post Properties' and hit retrieve button - No dialog box
2. [Posted to website](http://davidjohnmead.com/blog/2016/07/25/7026/) and no indication it was a reply (which was is normal)
3. Changed 'Post Kind' to reply and clicked publish which now displayed the content I was replying to

So that seems to work as it used to.

As soon as I create a new post and paste a Twitter URL into 'Post Properties', click retrieve, I get the blank dialog box. So maybe this is more a problem with 'Post Properties' part as at that point I hadn't typed any content in, or chosen the 'Post Kind'.
